src/CarbonAware/src/Configuration/EmissionsDataCacheConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.Extensions.Configuration;
+
+namespace CarbonAware.Configuration;
+
+internal class EmissionsDataCacheConfiguration
+{
+    public const string Key = "EmissionsDataCache";
+
+    public bool Enabled { get; set; }
+
+    public int ExpirationMin { get; set; }
+
+    public void AssertValid()
+    {
+        if(ExpirationMin < 0)
```

Comment:
Why do you allow `0`?  
As a cache, `0` should be error (or disable cache with warning message) I think.